tests/test_hand.py

```diff
@@ -55,3 +60,69 @@ def test_is_bust(
     assert not sixteen_hand.is_bust()
     assert not ace_hand.draw(ten).is_bust()
     assert sixteen_hand.draw(six).is_bust()
+
+
+def test_comparison_of_standard_hands() -> None:
+    hand_15 = test_hand(6, 9)

```

Comment:
Not sure I understand what you are testing here.

Your `test_starting_hand_value` should be enough to verify the total value of the hand. If it verifies that `test_hand(6, 9)` and `test_hand(2, 3)` gives the correct values 15 and 5, your assert statement on line 70 only asserts that 15 > 5 which will always be true.

Reply:
I want the hand class to be able to have it's own internal comparator. So for standard non-ace, non-bust hands, I want to verify it just compares their values directly.

Reply:
I don't know if that's overkill or not.

Reply:
In that case I think the test should then compare that the function doing the comparison outputs/executes the correct thing. Something like below maybe ? Correct me if I've misunderstood

```
def hand_compare(hand1, hand2):
    if hand1 > hand2:
        return True
    else:
        return False
    
def test_compare():
    assert hand_compare(3, 4) == False
   ```

Reply:
Is that not what I am doing? I was going write the hand_compare method using the __eq__, __gr__, __lt__ inbuilt methods